src/Altinn.App.Api/Controllers/TableOptionsController.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Altinn.App.Core.Features.Options;
+using Altinn.App.Core.Models;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Altinn.App.Api.Controllers
+{
+    /// <summary>
+    /// Represents the TableOptions API.
+    /// </summary>
+    [ApiController]
+    public class TableOptionsController: ControllerBase
+    {
+        private readonly IAppOptionsService _appOptionsService;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TableOptionsController"/> class.
+        /// </summary>
+        /// <param name="appOptionsService">Service for handling app options</param>
+        public TableOptionsController(IAppOptionsService appOptionsService)
+        {
+            _appOptionsService = appOptionsService;
+        }
+
+        /// <summary>
+        /// Api that exposes app related options
+        /// </summary>
+        /// <param name="optionsId">The optionsId</param>
+        /// <param name="language">The language selected by the user.</param>
+        /// <param name="queryParams">Query parameteres supplied</param>
+        /// <returns>The options list</returns>
+        [HttpGet]
+        [Route("/{org}/{app}/api/options/table/{optionsId}")]
```

Comment:
I suggest changing this to [Route("/{org}/{app}/api/lists/{listId}")]

It's a whole new concept and not related to options. It returns a list of objects which can be presented in a table, but don't have to.